terraform/bichard7-next-pnc-emulator.tf

```diff
@@ -0,0 +1,66 @@
+module "bichard7-next" {
```

Comment:
@bjpirt This key needs to be updated (it's already in use).

Reply:
Thanks - good old copy-paste!